ui/components/src/LineChart/LineChart.tsx

```diff
@@ -145,7 +147,10 @@ export function LineChart({
 
   const option: EChartsCoreOption = useMemo(() => {
     if (data.timeSeries === undefined) return {};
-    if (data.timeSeries === null || data.timeSeries.length === 0) return noDataOption;
+
+    // The "chart" `noDataVariant` is only used when the `timeSeries` is an
+    // empty array because a `null` value will throw an error.
```

Comment:
The `null` case should also be pretty rare if people are using TS because I'm pretty sure the types say it should be an array.